logger/syslogd/syslogd.go

```diff
@@ -82,6 +84,9 @@ func (h *handler) mainLoop() {
 		if m == nil {
 			break
 		}
+		if drain.GetDrain() != "" {
+			drain.SendToDrain(m.String(), drain.GetDrain())
```

Comment:
With each call to `.GetDrain()`, a new etcd client is created. We should be re-using the client instead of spawning a new connection.